sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/HDFSMetadataLogSuite.scala

```diff
@@ -209,14 +209,20 @@ class HDFSMetadataLogSuite extends SparkFunSuite with SharedSQLContext {
     }
 
     // Open and delete
-    val f1 = fm.open(path)
-    fm.delete(path)
-    assert(!fm.exists(path))
-    intercept[IOException] {
-      fm.open(path)
+    if (Utils.isWindows) {
+      fm.open(path).close()
+      fm.delete(path)
+      fm.delete(path) // should not throw exception
```

Comment:
I thought the tests are intended to delete twice (opened file). The first one removes it and the other one does nothing without any exception. The original test deletes twice.